<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PollerWidget</class>
 <widget class="QWidget" name="PollerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1230</width>
    <height>712</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_12">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>161</width>
       <height>200</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>250</width>
       <height>700</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_13">
      <property name="spacing">
       <number>14</number>
      </property>
      <item>
       <widget class="QLabel" name="label_18">
        <property name="text">
         <string>StartAddr</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="edtStartAddr">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>32</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_19">
        <property name="text">
         <string>Count</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="edtAddrCount">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="maximum">
         <number>32</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_21">
        <property name="text">
         <string>存储间隔(分钟)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="edtSaveTime">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>14</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_20">
        <property name="text">
         <string>采集速度(次/秒)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbxAcqSpan">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <item>
         <property name="text">
          <string>10次/秒</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5次/秒</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1次/秒</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>0.2次/秒</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>0.1次/秒</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnSetAddr">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>SetAddr</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSaveWave">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>SaveWave</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnClear">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>ClearWave</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWaveWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_6">
      <attribute name="title">
       <string>实时重量</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_13">
       <item>
        <widget class="QGroupBox" name="gbDevices">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(95, 95, 95);</string>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_14"/>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_7">
      <attribute name="title">
       <string>实时波形</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_15">
       <item>
        <widget class="QCustomPlot" name="rtplot" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
